SU-SWS/ace-sdssgryphon
# SDSS-228: focus for event cards

# READY FOR REVIEW

# Summary
- changed tab focus on event cards

# Review By (Date)
- 3/4

# Urgency
- Normal

# Steps to Test

1. Checkout this branch.
2. Clear caches.
3. copy down the main site and use this page to test `/about-school/news-and-events`
4. Verify when tabbing through those top event cards that you first go to the headline and then the topic.

# Affected Projects or Products
- SDSS Subtheme

# Associated Issues and/or People
[- SDSS-228](https://stanfordits.atlassian.net/browse/SDSS-228)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
